docs/backup-restore.md

```diff
@@ -60,6 +60,7 @@ To specify a transactionally-consistent restore point, please pause Scalar DL se
 ## Restore Backups of Ledger Databases
 
 To restore backups, you must follow the [Restore Backup](https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md#restore-backup) section.
+You must stop the Scalar DL Ledger service before restoring database backups and start the Scalar DL Ledger service after restoring the backups.
```

Comment:
Scalar DL Ledger service is working fine without restarting the Ledger service after backup restoration on DynamoDB.

But I would like to do this step to avoid the client's requests during the restoration.